dss/api/subscriptions.py

```diff
@@ -22,16 +26,151 @@
 
 
 def get(uuid: str, replica: str):
-    pass
+    owner = request.token_info['email']
+
+    es_client = ElasticsearchClient.get(logger)
+    id_exists = es_client.exists(index=DSS_ELASTICSEARCH_SUBSCRIPTION_INDEX_NAME,
+                                 doc_type=DSS_ELASTICSEARCH_SUBSCRIPTION_TYPE,
+                                 id=uuid)
+    if not id_exists:
+        return jsonify(dict(message="Subscription {} does not exist".format(uuid),
+                            exception="Placeholder",
+                            HTTPStatusCode=requests.codes.not_found)), requests.codes.not_found
+
+    response = es_client.get(index=DSS_ELASTICSEARCH_SUBSCRIPTION_INDEX_NAME,
+                             doc_type=DSS_ELASTICSEARCH_SUBSCRIPTION_TYPE,
+                             id=uuid)
+
+    source = response['_source']
```

Comment:
For Elasticsearch per se, and elasticsearch-py, '_source' is used very prominently and consistently in their docs. I am not aware of another option. The elasticsearch-dsl python library organizes its output differently and '_source' does not actually appear. The two output formats are different and each has its advantages and disadvantages.